brave-lists/clean-urls.json

```diff
@@ -73,6 +73,37 @@
             "ref_"
         ]
     },
+    {
+        "include": [
+            "*://*.ebay.com/itm/*",
+            "*://www.ebay.at/itm/*",
+            "*://www.ebay.ca/itm/*",
+            "*://www.ebay.co.uk/itm/*",
+            "*://www.ebay.com.au/itm/*",
+            "*://www.ebay.com.sg/itm/*",
+            "*://www.ebay.de/itm/*",
+            "*://www.ebay.es/itm/*",
+            "*://www.ebay.ie/itm/*",
+            "*://www.ebay.fr/itm/*",
+            "*://www.ebay.it/itm/*",
+            "*://www.ebay.ch/itm/*",
+            "*://www.ebay.nl/itm/*",
+            "*://www.ebay.pl/itm/*"
+        ],
+        "exclude": [
+
+        ],
+        "params": [
+            "media",
+	    "mkcid",
+            "mkevt",
+            "ssspo",
+            "ssrc",
+	    "ssuid",
+            "var",
```

Comment:
That's the only one I feel uneasy about. In the example URL we have, it's not set to anything and so it's not clear what it's for or whether or not it's useful. Also, it is **very** generic.

I would leave that one out for now.